package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chainsafe/persistent-merkle-tree",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
added new feature -> increment minor version?

Reply:
I'm considering 0.x.y to be initial development, incrementing minor version on breaking change.